here-naksha-app-service/src/test/resources/unit_test_data/Auth/validJwtClaims.json

```diff
@@ -0,0 +1,15 @@
+{
+  "appId": "web-client-app-id",
+  "userId": "my-user-id",
+  "urm": {
+    "naksha": {
+      "readFeatures": [
```

Comment:
also add "createFeatures" with empty attribute map, to allow "testDummyModeNoJWTAppIdAuthorExistsInXyzNamespace" function to create feature.